WalletWasabi.Documentation/Guides/DeterministicBuildGuide.md

```diff
@@ -1,68 +1,90 @@
+Now, we will attempt to verify the binaries you have just compiled with the officially distributed binaries on https://wasabiwallet.io website. Please download those packages from the website, you should see the following files in your File Explorer:
 
 ![](https://i.imgur.com/aI9Kx0c.png)
 
-## Windows
+### Windows
 
-After you installed Wasabi from the `.msi`, it will be in `C:\Program Files\WasabiWallet` folder. You can compare it with your build:
-
-```sh
-git diff --no-index win7-x64 "C:\Program Files\WasabiWallet"
-```
+* Install Wasabi using `Wasabi-<version>.msi` file. It will install to `C:\Program Files\WasabiWallet` directory.
+* Start `cmd` or Powershell and navigate to the `dist` directory.
+* Execute the following command:
+  ```sh
+  git diff --no-index "win7-x64" "C:\Program Files\WasabiWallet"
+  ```
+  and make sure that there is **NO** difference reported by the command.
 
-## Linux && OSX
+### Linux & macOS
 
-You can use the Windows Subsystem for Linux to verify all the packages in one go. At the time of writing this guide we provide a `.tar.gz` and a `.deb` package for Linux and .dmg for OSX.  
+You can use the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/) to verify all the packages in one go. At the time of writing this guide we provide `.tar.gz` and `.deb` packages for Linux and `.dmg` package for macOS.  
 Install the `.deb` package and extract the `tar.gz` and `.dmg` packages, then compare them with your build.
 
-After installing WSL, just type `wsl` in explorer where your downloaded and built packages are located.
+After [installing WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10), just type `wsl` in File Explorer where your downloaded and built packages are located.
 
 ![](https://i.imgur.com/yRUjxvG.png)
 
-### .deb
+#### .deb
 
 ```sh
 sudo dpkg -i Wasabi-1.1.6.deb
 git diff --no-index linux-x64/ /usr/local/bin/wasabiwallet/
 ```
 
-### .tar.gz
+#### .tar.gz
 
 ```sh
 tar -pxzf Wasabi-1.1.6.tar.gz
 git diff --no-index linux-x64/ Wasabi-1.1.6
 ```
 
-### .dmg
+*There could be warnings reagarding SOS_README.md that it differs in line endings. That is a text file and it has no effect on the running software.*
+
+#### .dmg
+
+According to Apple documentation, the signature that is used to ensure the integrity of the software is added into the binary itself - so it will manipulate the content of the files.
+
+> If the code is universal, the object code for each slice (architecture) is signed separately. This signature is stored within the binary file itself.
+
+[Source](https://developer.apple.com/library/archive/documentation/Security/Conceptual/CodeSigningGuide/AboutCS/AboutCS.html#//apple_ref/doc/uid/TP40005929-CH3-SW3)
+
+According to this, it is impossible to have both deterministic build and code signature on macOS. macOS Gatekeeper won't let you run software without it - thus code signature is prior to the deterministic build. 
```

Comment:
> thus code signature is prior to the deterministic build.

Prior? What do you mean?

Reply:
That it is more important than the det builds

Reply:
Do you have a better word here?

Reply:
```suggestion
According to this, it is impossible to have both deterministic build and code signature on macOS. macOS Gatekeeper won't let you run software without it - thus Wasabi only applies code signature, but no deterministic build for macOS. 
```

Reply:
> That it is more important than the det builds
> Do you have a better word here?

`prior to` means `before`. You have probably meant to use word `priority`. I'm not sure. Hence the question to make sure, it's correct.

I like Max's wording.